tools/packaging/kata-deploy/helm-chart/kata-deploy/templates/kata-rbac.yaml

```diff
@@ -30,6 +30,9 @@ rules:
 - apiGroups: ["nfd.k8s-sigs.io"]
   resources: ["nodefeaturerules"]
   verbs: ["create", "get", "list", "watch"]
+- apiGroups: ["apps"]
+  resources: ["daemonsets"]
+  verbs: ["get", "list", "watch"]
```

Comment:
For the sake of consistency, this should also be updated here:  https://github.com/kata-containers/kata-containers/blob/657c4406cdb5b26b93ee1fce08c5dc20d877cc3f/tools/packaging/kata-deploy/helm-chart/kata-deploy/templates/post-delete-job.yaml#L40-L42